CHANGELOG.md

```diff
@@ -4,6 +4,31 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.5.0
+### Added
+* Agent: Include `LD_LIBRARY_PATH` if and only if set by command. [#1933](https://github.com/microsoft/onefuzz/pull/1933)
```

Comment:
```suggestion
* Agent: Include `LD_LIBRARY_PATH` in shared library dependency check if and only if set by command. [#1933](https://github.com/microsoft/onefuzz/pull/1933)
```
Should've suggested this wording before (original doesn't make sense without the rest of the PR text).